src/map/js/helpers/LayersHelper.js

```diff
@@ -665,22 +665,29 @@ let LayersHelper = {
       // We are hiding and showing the layer to avoid calling the service multiple times.
       viirs.hide();
       const layaDefs = [];
+
       switch(optionIndex) {
         case 0: //past 24 hours
           viirs.url = shortTermServices.viirs24HR.url;
           viirs._url.path = shortTermServices.viirs24HR.url;
           viirs.setVisibleLayers([shortTermServices.viirs24HR.id]);
+          viirs.dynamicLayerInfos[0].id = shortTermServices.viirs24HR.id;
+          viirs.dynamicLayerInfos[0].source.mapLayerId = shortTermServices.viirs24HR.id;
           break;
         case 1: //past 48 hours
           viirs.url = shortTermServices.viirs48HR.url;
           viirs._url.path = shortTermServices.viirs48HR.url;
           viirs.setVisibleLayers([shortTermServices.viirs48HR.id]);
+          viirs.dynamicLayerInfos[0].id = shortTermServices.viirs48HR.id;
+          viirs.dynamicLayerInfos[0].source.mapLayerId = shortTermServices.viirs48HR.id;
           break;
         case 2: //past 72 hours
           viirs.url = shortTermServices.viirs7D.url;
           viirs._url.path = shortTermServices.viirs7D.url;
           viirs.setVisibleLayers([shortTermServices.viirs7D.id]);
           layaDefs[shortTermServices.viirs7D.id] = `Date > date'${new window.Kalendae.moment().subtract(3, 'd').format('YYYY-MM-DD HH:mm:ss')}'`;
+          viirs.dynamicLayerInfos[0].id = shortTermServices.viirs7D.id;
+          viirs.dynamicLayerInfos[0].source.mapLayerId = shortTermServices.viirs7D.id;
           break;
         case 3: //past 7 days
```

Comment:
@Anthony-Calderaro Is there a reason that this `case` doesn't need this **id** & **source.mapLayerId** property updates like the others?

Reply:
This case should have a maplayerid and id, so I added a commit to include it.